pkg/os/filesystem/api.go

```diff
@@ -78,6 +80,29 @@ func (filesystemAPI) Rmdir(path string, force bool) error {
 	return os.Remove(path)
 }
 
+// RmdirContents removes the contents of a directory with `os.RemoveAll`
+func (filesystemAPI) RmdirContents(path string) error {
+	dir, err := os.Open(path)
```

Comment:
https://stackoverflow.com/a/52685448

a reference of implementation

Reply:
I took the same implementation as LVP for linux